01-wasmcloud_basics/wash_intro/ctl.md

```diff
@@ -0,0 +1,99 @@
+ hostcore.arch                                      x86_64
+ hostcore.osfamily                                  unix
+                     No actors found
+
+ Provider ID            Link Name                  Image Reference
+ VDHPK<..SNIP..>SVREP   default                    N/A
+ VAHNM<..SNIP..>7S6Z2   __wasmcloud_lattice_cache  N/A
+```
+
+> Note: The output has been condensed for the tutorial. Your tokens will be 56 characters long and differ from the ones displayed here.
+
+##### Important Fields
+
+In this example, you'll notice in the middle of the output `No actors found`. So far, we do not have any actors running in this environment. Next, we will start the `echo` actor and you will be able to see it here.
+
+Below the actor output is running provider information.  
+[TODO: Describe what the 2 items in the provider section are for.]
+
+### wash ctl start actor
+
+In order to start an actor (or provider), we must envoke the `wash ctl start` command. This is the first step to running our business logic in the `wasmcloud` ecosystem.
+
+#### Example
+
+`ctl start actor wasmcloud.azurecr.io/echo:0.2.1`{{execute}}
+
+> Note: Since we have not added our `table_tennis` actor to the registry yet, we will start the `echo` actor that is in the `wasmcloud` container registry.
+
+The output will confirm that we have started an actor. Now, we will be able to check out inventory again and see an actor running.
+
+`ctl get inventory <HOST-ID>`{{copy}}
+
+> Note: You can use the up arrow within the wash REPL to re-run commands from your history.
+
+```bash
+<SNIP>
+
+Actor ID                 Image Reference
+MBCFOP<..SNIP..>HD3WD5   wasmcloud.azurecr.io/echo:0.2.1
+
+<SNIP>
+```
+
+### wash ctl start provider
+
+Just like starting an actor, you can start a provider with the `wash ctl start` command. To know what provider we should start for the echo actor, we can inspect it and see what capabilities it has.
+
+> Note: Remember from the previous screen, you can use `wash inspect` to see what capabilities an actor has. The `echo` actor has the HTTP Server capability.
+
+#### Example
+
+`ctl start provider wasmcloud.azurecr.io/httpserver:0.12.1`{{execute}}
+
+Once again, you will see the output that your provider started successfully and you can verify it by checking the inventory.
+
+### wash ctl link
+
+Now that you have the actor and the provider running, you need to tell them to communication with con another. This is done in the linking process via `wash ctl link`. We are going to tell the `wasmcloud` runtime that we want our actor talking with out provider, and give it a few settings to do to.
+
+> Note: In this example, you will need both the ACTOR ID and the PROVIDER ID that is provided from the inventory output. If you run the `get inventory` command, you will have that information in the output window while you are typing the link command.
+
+#### Example
+
+`ctl link <ACTOR-ID> <PROVIDER-ID> wasmcloud:httpserver PORT=8080`{{copy}}
```

Comment:
You can replace `<PROVIDER-ID>` here with `VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M`, it won't change